ASWF/meetings/working_groups/configs/2020-11-03.md

```diff
@@ -0,0 +1,147 @@
+      spline to keep from running off edge of LUT, so those would need a Lut1D.
+    - Mark: Talked before about making multiple configs, with customizable 
+      generators. Think some IDTs would not make the cut, but could be in 
+      another version. ARRI could be in there since it's common.
+    - J: EI 800 would make sense, but not sure all are needed.
+    - Doug: Current ACES config has a bunch of these color spaces so need to 
+      decide about compatibility.
+    - J: Need to figure out which flavor we publish. One complaint we have 
+      heard is the config is unwieldy because of so many color spaces.
+    - Mark: Not sure how many need to be builtin transforms as well. Could 
+      include LUTs and matrices in some. Could be example for implementation.
+    - Michael: We'll need the transforms that make ACES ACES for the reference 
+      config. Others can be for the studio config.
+    - J: What determines which are included?
+    - Mark: We decide with community feedback.
+    - Doug: Thomas' graph will illustrate that. Another area where manual 
+      intervention is needed. Don't need all ARRI IDTs.
+    - Mark: Already filtering if not EI 800.
+    - J: EI 800 is base one anyway, so good mandate.
+    - Doug: Other thing about the IDTs. ACES repo doesn't include CTL for RED, 
+      Canon, etc. Also need to decide about those. In current ACEs config some 
+      work was done to track down that stuff and create transforms.
+    - Michael: Could cut off stuff not in aces-dev repo.
+    - Doug: Need some RED and Canon, and there are CSC transforms for those. 
+      Maybe don't go beyond that. Some needed discussion around what those 
+      transform are. Was hoping Thomas' work was to produce list of proposed 
+      transform IDs to use as builtin transform names. Think thats one of the 
+      next steps. Make spreadsheet of specific list of transforms. Which have 
+      CTL and which don't, and assign someone to figure out the latest and 
+      greatest collection.
+    - Mark: Lot of noise in output from generator, filtering through 
+      relationships with various rules. Not sure how much of that snuck into 
+      graph. Not sure how many intermediates we want in there too.
+    - Doug: Think it will help. Discussion that needs more community input. 
+      What is in the config and what are the strings.
+    - Michael: Can put all transforms in spreadsheet and show which are 
+      excluded and why.
+    - Mark: Can have column for what is included.
+    - **TODO**: Mark will use generator to create a spreadsheet for discussion
+      of which transforms are included in reference config.
+    - **TODO**: Doug will create example of builtin output transform in OCIO.
+
+* ACES reference config goal
+    - Michael: Shall we set a goal to have the reference config done by OCIO 
+      v2 (non-RC) release? Would be good to ship an example if possible.
```

Comment:
That would be a good idea and quite coincidentally: https://lists.aswf.io/g/ocio-dev/message/2037